### DISCUSSION TITLE
nodeID, descripton and label of "GR-Procedure-PlaceOfPerformance"

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#899] [Ideas]
Why do nodeID, description and lable each contain the addition "Additional information" at procedure level?

For example subtyp 16:
 
        "id" : "GR-Procedure-PlaceOfPerformance",
        "contentType" : "group",
        "nodeId" : "ND-ProcedurePlacePerformanceAdditionalInformation",
        "displayType" : "GROUP",
        "description" : "Additional Information about the location where the Procurement, for the Procedure as a whole, should take place",
        "_label" : "group|name|ND-ProcedurePlacePerformanceAdditionalInformation",

**Comment by YvesJo** [comment]
Node ID and description are generally based on their content:

- ND-ProcedurePlacePerformanceAdditionalInformation contains a description providing further (i.e. additional) information about the place of performance, while
- ND-ProcedurePlacePerformance contains the detailed place of performance (e.g. city, street ...)

Label ID is composed of the Node ID

**Reply by JanEmde** [reply]
However, I can find "ND-ProcedurePlacePerformance" only in the view templates. The notice type definitions and the translation files only contain "ND-ProcedurePlacePerformanceAdditionalInformation".

**Reply by YvesJo** [reply]
Nodes are defined once as part of the nodes and fields to describe the information global structure. They are reused for further operations.
NTDs are defined for the UI and must explicitly refer repeatable nodes so that the user may add extra occurrences of that node when needed ("ND-ProcedurePlacePerformanceAdditionalInformation" is repeatable). On the UI, information is put under groups to have it together, not all nodes are necessary, also some group may just exist to improve the presentation (kind of logical grouping, "Display Group"); When information is in a Group associated to a node that do not correspond to the parent group, i is however important that that node is associated to an ancestor node.
For the view-template, nodes are meanly used as operation contexts and it is not mandatory to have them all even if they are repeatable, it may be sufficient to process based on an ancestor.

**Comment by JanEmde** [comment]
I get that. But the current implementation (SDK 1.10) means that the group at procedure level is currently shown as "Additional information about place of performance".

Although the group should only be called Place of Performance according to the translation file:
entry key="group|name|GR-Procedure-PlaceOfPerformance">Place of performance</entry

Screenshot from the German Redaktionssystem:
![grafik](https://github.com/OP-TED/eForms-SDK/assets/144460224/6e580832-eec9-40a6-a3bc-9b04b8ebf750)

**Reply by YvesJo** [reply]
indeed, the improper node is associated to the group at procedure level, this will get fixed in SDK 1.12

**Reply by JanEmde** [reply]
Thanks!
